src/contexts/auth-context/authContext.tsx

```diff
@@ -0,0 +1,76 @@
+import { createContext, useEffect, useState, FC, useContext } from "react";
+import useLocalStorage from "../../custom-hooks/useLocalStorage";
+import { UserType, UsersType, authContextType } from "./authContext.types";
+import fakeAuthAPI from "../../helpers/fakeAuth";
+// import { createContext, FC, useContext, useReducer } from "react";
+import { fakeAuthResponseType } from "../../helpers/fakeAuth";
+
+const initialContext = {
+  isUserLoggedIn: false,
+  loginWithCredentials: () => null,
+  signUpUser: () => null,
+  currentUser: null,
+};
+
+export const authContext = createContext<authContextType>(initialContext);
```

Comment:
consistent naming with context types can be used. Here can be AuthContext
and 'type' word can be avoided in types names